src/System Application/App/Email/src/RateLimit/EmailRateLimit.Table.al

```diff
@@ -44,6 +44,17 @@ table 8912 "Email Rate Limit"
                     Error(ConcurrencyLimitErrLbl);
             end;
         }
+        field(6; "Max. Retry Limit"; Integer)
+        {
+            DataClassification = SystemMetadata;
+            InitValue = 1;
+
+            trigger OnValidate()
+            begin
+                if (Rec."Max. Retry Limit" < 1) or (Rec."Max. Retry Limit" > 20) then
+                    Error(MaxRetryLimitErrLbl);
```

Comment:
Why don't you instead/additionally work with properties `MinValue` and `MaxValue`?